googleapiclient/discovery.py

```diff
@@ -82,6 +82,10 @@
 VARNAME = re.compile('[a-zA-Z0-9_-]+')
 DISCOVERY_URI = ('https://www.googleapis.com/discovery/v1/apis/'
                  '{api}/{apiVersion}/rest')
+V1_DISCOVERY_URI = DISCOVERY_URI
+V2_DISCOVERY_URI = ('https://{api}.googleapis.com/$discovery/rest?'
```

Comment:
If you line-wrap immediately after the opening parenthesis can you fit your entire string on the following line without breaking it?

Implicit string concatenation is one of Python's most dangerous problems and one should always try to avoid it.